conda-store-server/conda_store_server/server/views/ui.py

```diff
@@ -68,6 +68,30 @@ def ui_list_environments(
     return templates.TemplateResponse("home.html", context)
 
 
+@router_ui.get("/coatl")
+def ui_list_environments(
```

Comment:
Change the name of this method. It will collide with the `/` route.

Reply:
whoops. ill rename it to `ui_coatl`